src/CitizenProject/CitizenProjectBroadcast.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace AppBundle\CitizenProject;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\CitizenProject;
+use AppBundle\Mailer\MailerService;
+use AppBundle\Mailer\Message\CitizenProjectApprovedSummaryMessage;
+use AppBundle\Mailer\Message\Message;
+use Psr\Log\LoggerInterface;
+use Twig\Environment;
+
+class CitizenProjectBroadcast
+{
+    private $mailer;
+    private $twig;
+    private $logger;
+
+    public function __construct(MailerService $mailer, Environment $twig, LoggerInterface $logger)
+    {
+        $this->mailer = $mailer;
+        $this->twig = $twig;
+        $this->logger = $logger;
+    }
+
+    public function handle(array $citizenProjects, Adherent $adherent): void
```

Comment:
Imo, the broadcaster should be responsible for querying the projects, it then only needs one adherent to broadcast to.

Reply:
Should be named `broadcastNearbySummariesToAdherent()`.